alfasim_sdk/types.py

```diff
@@ -16,7 +16,7 @@ class ALFAsimType:
 
 @attr.s(kw_only=True)
 class TracerType(ALFAsimType):
-    type = attrib(default="tracer")
+    container_type = attrib(default="TracerModelContainer")
```

Comment:
```suggestion
    _CONTAINER_TYPE = "TracerModelContainer"
```

This seems like it should be a class attribute instead.